### PR TITLE
Add postmortem of utility score bug

### PR DESCRIPTION
**Pull request by ChewingGlass**

### CODE REVIEW

File: devblog/2023-10-18-utility-score-bug-post-mortem.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: MOBILE Utility Score Bug Post Mortem (Pre HIP-88)
+date: 2021-10-18
+hide_table_of_contents: true
+authors: [cokes518]
```
Comment (jthiller) [inline devblog/2023-10-18-utility-score-bug-post-mortem.mdx]: Let's add you to the authors table. I can push it to this branch.

Comment (ChewingGlass) [reply devblog/2023-10-18-utility-score-bug-post-mortem.mdx]: Oh oops, sorry had copied this from another one

**Comment by jthiller** [comment]
Rendered View: https://deploy-preview-1649--helium-docs.netlify.app/devblog/2023/10/18/utility-score-bug-post-mortem

**Review by waveform06 — DISMISSED** [review]

File: devblog/2023-10-18-utility-score-bug-post-mortem.mdx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+hide_table_of_contents: true
+authors: [noah]
+---
+
+&nbsp;
+
+# MOBILE Utility Score Bug Post Mortem (Pre HIP-88)
+
+This post-mortem describes the events leading to an incorrect A portion of the Utility Score for the
+MOBILE SubNetwork, what happened, and what the team is doing moving forward.
```
Comment (abhay) [inline devblog/2023-10-18-utility-score-bug-post-mortem.mdx]:
```suggestion
This post-mortem describes the events leading to an incorrect `A` portion of the DAO Utility Score for the
MOBILE subnetwork, what happened, and what the team is doing moving forward.
```

File: devblog/2023-10-18-utility-score-bug-post-mortem.mdx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+<!--truncate-->
+
+## Timeline
+
+On July 21st, 2023, MOBILE Subscriber Mapping went live. This introduced a new rewardable entity
+into the MOBILE ecosystem.
```
Comment (abhay) [inline devblog/2023-10-18-utility-score-bug-post-mortem.mdx]:
```suggestion
On July 21st, 2023, MOBILE Subscriber Mapping launched. This introduced a new rewardable entity
into the MOBILE ecosystem.
```

File: devblog/2023-10-18-utility-score-bug-post-mortem.mdx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+Helium Oracles are powered by a data pipeline that runs over Proof of Coverage and network usage
+data. After generating rewards shares, the lifetime rewards for each entity are calculated and
+inserted into a Postgres database. When a user initiates a claim, the Oracle service then reads the
+values from the database and signs the claim transaction to set the lifetime rewards for that entity
+on-chain.
```
Comment (abhay) [inline devblog/2023-10-18-utility-score-bug-post-mortem.mdx]:
```suggestion
Helium Oracles are powered by a data pipeline that runs over Proof-of-Coverage and network usage
data. After generating rewards shares, the lifetime rewards for each entity are calculated and
inserted into a Postgres database. When a user initiates a claim, the Oracle service then reads the
values from the database and signs the claim transaction to set the lifetime rewards for that entity
on-chain.
```

**Review by abhay — COMMENTED** [review]
LGTM. Small edits.

**Review by jthiller — APPROVED** [review]

**Pull request merged**